packages/office-ui-fabric-react/src/components/FocusTrapZone/examples/FocusTrapZone.Box.Click.Example.tsx

```diff
@@ -2,9 +2,18 @@ import * as React from 'react';
 
 import { FocusTrapZone } from 'office-ui-fabric-react/lib/FocusTrapZone';
 import { Link } from 'office-ui-fabric-react/lib/Link';
-import { TextField } from 'office-ui-fabric-react/lib/TextField';
+import { TextField, ITextFieldStyles } from 'office-ui-fabric-react/lib/TextField';
 import { Toggle, IToggle } from 'office-ui-fabric-react/lib/Toggle';
-import { Stack } from 'office-ui-fabric-react/lib/Stack';
+import { Stack, IStackStyles } from 'office-ui-fabric-react/lib/Stack';
+import { memoizeFunction } from '@uifabric/utilities';
```

Comment:
Same thing for all imports of this
```suggestion
import { memoizeFunction } from 'office-ui-fabric-react/lib/Utilities';
```